eth-contracts/contracts/staking/Staking.sol

```diff
@@ -95,12 +95,9 @@ contract Staking is RegistryContract, StakingInterface {
         requireIsInitialized();
         require(
             msg.sender == registry.getContract(delegateManagerKey),
-            "slash only callable from DelegateManager"
+            "Only callable from DelegateManager"
         );
 
-        // unstaking 0 tokens is not allowed
-        require(_amount > 0, ERROR_AMOUNT_ZERO);
-
```

Comment:
why remove? would make sense to remove if delegateManager.slash enforced this already, but it doesn't. (although it should)

Reply:
_burnFor restricted